ui/lib/core/addon/components/overview-card.hbs

```diff
@@ -20,6 +20,7 @@
         @iconPosition="trailing"
         @text={{@actionText}}
         @route={{@actionTo}}
+        @model={{@actionModel}}
```

Comment:
I added this as a minimal change for this chunk of work, but in the future I'd like to refactor this component so that it's contextual